src/app/containers/ChartbeatAnalytics/index.test.jsx

```diff
@@ -120,35 +118,31 @@ describe('Charbeats Analytics Container', () => {
       {},
     );
     expect(testUtils.getConfig).toHaveBeenCalledTimes(1);
-    expect(tree).toMatchSnapshot();
+    expect(container.firstChild).toMatchSnapshot();
   });
 
   it('should return null when toggle is disbaled for live', () => {
-    const tree = renderer
-      .create(
-        <ContextWrap platform="canonical" pageType="article" origin="bbc.com">
-          <ChartbeatAnalytics data={frontPageData} />
-        </ContextWrap>,
-      )
-      .toJSON();
-
-    expect(tree).toMatchSnapshot();
+    const { container } = render(
+      <ContextWrap platform="canonical" pageType="article" origin="bbc.com">
+        <ChartbeatAnalytics data={frontPageData} />
+      </ContextWrap>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
judging by the title of this test and the pointless null snapshots here https://github.com/bbc/simorgh/blob/a7cc3152e610fbd0de4f34ff18657f47f596aa9e/src/app/containers/ChartbeatAnalytics/__snapshots__/index.test.jsx.snap I would change the assertion to:

`expect(container.firstChild).toBeNull();`